src/main/scala/net/snowflake/spark/snowflake/CloudCredentialsUtils.scala

```diff
@@ -26,35 +26,9 @@ import net.snowflake.client.jdbc.internal.amazonaws.auth.{
   InstanceProfileCredentialsProvider
 }
 import net.snowflake.client.jdbc.internal.microsoft.azure.storage.StorageCredentialsSharedAccessSignature
-import net.snowflake.spark.snowflake.Parameters.MergedParameters
 import org.apache.hadoop.conf.Configuration
-import org.apache.spark.sql.SQLContext
 
 private[snowflake] object CloudCredentialsUtils {
```

Comment:
Some explain for reviewers:
Three functions are removed because:
1. They are not used.
2. They looks to be used to specify info for external stage for COPY command.
     Even with existing external stage, these info is not needed. 
     Actually, We create a named stage for the external stage,
     and we use the named stage directly in the COPY command.